contracts/src/Teleporter/upgrades/README.md

```diff
@@ -2,28 +2,29 @@
 
 ## Overview
```

Comment:
Can you add a paragraph explaining the `getLatestVersion` method? Specifically, we should make it clear that this returns the numerically largest version number registered in the contract. The language can be a bit confusing, since "latest" may be interpreted as the most recent Teleporter version registered, not necessarily the highest number version.